SU-SWS/stanford_basic
# D8CORE-445: adding _line_length.scss

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-445_
This limits the number of characters per line to around 85.

# Needed By (Date)
- Sprint end

# Urgency
- Not Urgent

# Steps to Test
- Checkout this branch
- Compile the sass `npm run publish`
- Verify that h1-h6, p, and li elements do not exceed 85 characters

# Affected Projects or Products
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-445

## Related PRs

## More Information

## Folks to notify
@kerri-augenstein

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
